spec/models/vm_scan/dispatcher_get_eligible_proxies_for_job_spec.rb

```diff
@@ -8,7 +8,7 @@
       allow_any_instance_of(MiqServer).to receive_messages(:is_vix_disk? => true)
 
       # Support old style class methods or new instance style
-      @jpd = JobProxyDispatcher.respond_to?(:get_eligible_proxies_for_job) ? JobProxyDispatcher : JobProxyDispatcher.new
+      @jpd = VmScan::Dispatcher.respond_to?(:get_eligible_proxies_for_job) ? VmScan::Dispatcher : VmScan::Dispatcher.new
```

Comment:
This is weird, but don't we know ahead of time if it responds to it or not? (I know you were just doing find/replace, so more just asking the question for the future)

Reply:
Yeah this was just a copy&paste I don't know why it wouldn't know if it was testing a class method vs an instance method